changelog/1813.breaking.rst

```diff
@@ -0,0 +1 @@
+Changed the gyroradius function so that it takes relativistic effects into account.
```

Comment:
```suggestion
Changed the `~plasmapy.formulary.lengths.gyroradius` function so that it takes relativistic effects into account.
```
... to add a link to `gyroradius` in the docs.